frontends/ol-components/src/components/RoutedDrawer/RoutedDrawer.tsx

```diff
@@ -111,15 +121,16 @@ const RoutedDrawer = <K extends string, R extends K = K>(
               params: childParams as Record<K, string>,
               closeDrawer: setOpen.off,
             })}
-          <ActionButton
-            style={closeSx}
-            variant="text"
-            size="medium"
-            onClick={setOpen.off}
-            aria-label="Close"
-          >
-            <CloseIcon />
-          </ActionButton>
+          {!hideCloseButton && (
+            <CloseButton
+              variant="text"
+              size="medium"
+              onClick={setOpen.off}
+              aria-label="Close"
+            >
+              <CloseIcon />
+            </CloseButton>
+          )}
```

Comment:
The only instance we have of `RoutedDrawer` sets `hideCloseButton={true}`... Can we skip changing the close button style here?